core/controllers/pages.py

```diff
@@ -26,11 +27,16 @@ class SplashPage(base.BaseHandler):
 
     def get(self):
         """Handles GET requests."""
+        c_value = self.request.get('c')
+        if not c_value:
+            c_value = 's%d' % random.randrange(4)
+            self.redirect('/splash?c=%s' % c_value)
```

Comment:
Why do you display a random splash page? It could be confusing to users to see a different splash page each time they visit, so perhaps displaying just one consistently is best? (I'm assuming that the purpose of having the different pages is to be able to show people different splash pages through the link?)